src/patternfly/components/Brand/brand.scss

```diff
@@ -1,4 +1,14 @@
-// stylelint-disable
+$pf-c-brand--breakpoint-map: build-breakpoint-map();
+
 .pf-c-brand {
+  --pf-c-brand--Width--base: auto;
+  --pf-c-brand--Height--base: auto;
+
+  display: flex;
```

Comment:
This will be breaking with the existing component since it will make the inline `img` have block formatting. Maybe just limit this to `.pf-m-picture`?